src/shadertools/modules/fp64/fp64-arithmetic.glsl.js

```diff
@@ -39,7 +39,7 @@ vec2 split(float a) {
 }
 #endif
 
-#if defined(NVIDIA_EQUATION_WORKAROUND) || defined(INTEL_EQUATION_WORKAROUND)
+#if defined(NVIDIA_FP64_WORKAROUND) || defined(INTEL_FP64_WORKAROUND)
```

Comment:
Idea: How about changing the name of the flag to `FP64_COMPILER_OPTIMIZATION_WORKAROUND`:

Reasons:
* We'd use a single "workaround flag" rather than multiple GPU flags, since the code here is not really GPU specific, it is a fairly general workaround. 
* The proposed name would be more descriptive of what this workaround actually does, improving readability.
* The new name uses the name of the module as a prefix which is usually good practice and matches conventions in other shader modules

Also:
* With this new convention, we'd have the option of just leaving it enabled for any GPU we don't recognize, rather than just enabling for those that we do. 
* Are there any ill effects of enabling this except for a small performance hit?

In the balance of things, I assume we'd much rather avoid rendering issues on new GPUs rather than take the risk that we have a small perf hit.